apps/docs/content/components/RiverAccordion.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: River Accordion
+description: Use the river accordion to create collapsible content panels with associated visuals.
```

Comment:
Being pedantic but should this be `to create expandable content panels with associated visuals.` instead of `collapsible` because you a) can't collapse an open panel and if you're presenting a visual you're expanding not collapsing.